types/core/index.d.ts

```diff
@@ -240,6 +240,11 @@ export interface IParsingOptions {
     | false;
 }
 
+export interface ChartActiveElement {
```

Comment:
Still not a TS user, so this might be against the principles, but I'd tend to make this a bit more generic declaration (name) and use it in the tooltip too. Also I'd extend the same declaration with the `element` addition as the result of `getActiveElements`

Reply:
Sure, I'll make that a bit more generic. Good catch on `element` as well. Initially during dev I had the API take all 3 as input, but asking users to go lookup the element seemed like a pain and I missed updating the return type.